multicall/multicall.py

```diff
@@ -19,7 +18,7 @@ def __init__(self, calls: List[Call], _w3=None, block_id=None, require_success:
 
     def __call__(self):
         if self.require_success is True:
-            multicall_map = MULTICALL_ADDRESSES
+            multicall_map = MULTICALL_ADDRESSES if self.w3.eth.chainId in MULTICALL_ADDRESSES else MULTICALL2_ADDRESSES
```

Comment:
`chain_id` pls, the camelCase will be deprecated

Reply:
Fix'd